### PR TITLE
Translations el fr

### PR DESCRIPTION
**Pull request by brynyShift72**
Translations Updates for el_EL, es_ES, es_MX, fi_FI and fr_FR.

### CODE REVIEW

**Comment by stajs** [comment]
Can you resolve the conflicts first?

**Comment by smitsanghvi21** [comment]
@brynyShift72 you can click the resolve conflicts button and resolve it through web editor instead of vs code.

File: site/es_ES.all.json [hunk 1]
```diff
@@ -482,120 +482,116 @@
...
+"changepincode_modal_pincode2": { "other": "Confirma tu nuevo PIN de 4 dígitos" },
+"changepincode_modal_submit": { "other": "Establecer PIN" },
+"changepincode_modal_error_wrong_password": { "other": "Esa contraseña es incorrecta." },
+
+"user_set_pin_intro": { "other": "Necesitas un PIN para {{.Action}} contenido restringido"},
+"user_error_wrong_pin_reset": { "other": "Haga clic aquí para restablecer su PIN" },
 "user_error_too_many_pin_errors_help_page_link": { "other": "" },
-"shopping_error_age_restricted": { "other": "This content is age restricted." },
+"shopping_error_age_restricted": { "other": "Este contenido tiene restricción de edad." },
 "shopping_error_close_button": { "other": "OK" },
```
Comment (stajs) [inline site/es_ES.all.json:546]: lol `OK` is universal eh? Might want to just check, e.g. for Ukranian it got translated to `гаразд`.

File: site/es_MX.all.json [hunk 2]
```diff
@@ -421,153 +421,150 @@
...
+  "wcag_aria_label_bundle_items": { "other": "Paquete de artículos" },
+  "wcag_aria_label_carousel": { "other": "Carrusel" },
+  "wcag_aria_label_page_collection": { "other": "Colección de páginas" },
+  "wcag_aria_label_collection_list": { "other": "Lista de colecciones" },
+  "wcag_aria_label_collection_slider": { "other": "Control deslizante de colección" },
+  "wcag_aria_label_wishlist": { "other": "Lista de deseos" },
+  "wcag_aria_label_facebook": { "other": "Compartir en Facebook" },
+  "wcag_aria_label_twitter": { "other": "Compartir en Twitter" },
+  "wcag_aria_label_linkedin": { "other": "Compartir en LinkedIn" },
+  "wcag_aria_label_letterboxd": { "other": "Ver en Letterboxd" },
```
Comment (stajs) [inline site/es_MX.all.json:445]: I think you've done this better than I did - I translated the company names too which does come out different.

File: site/es_MX.all.json [hunk 3]
```diff
@@ -421,153 +421,150 @@
...
-  "shopping_payment_method_header": { "other": "Payment methods" },
-  "shopping_payment_method_credit_card": { "other": "Credit card" },
-  "shopping_payment_method_giropay": { "other": "Giropay" },
-  "shopping_error_stripe_unknown_error": { "other": "Something went wrong with this payment. Please try again later." },
-  "shopping_error_payment_complete_failed": { "other": "The payment was declined. Close this window and try again." },
+  "shopping_error_plan_already_owned": { "other": "Ya tienes este plan." },
+  "shopping_error_plan_expired": { "other": "Este plan ya no está disponible." },
+  "shopping_payment_method_header": { "other": "Métodos de pago" },
+  "shopping_payment_method_credit_card": { "other": "Tarjeta de crédito" },
+  "shopping_payment_method_giropay": { "other": "Giropay (Alemania)" },
```
Comment (stajs) [inline site/es_MX.all.json:451]: And this is smarter than what I did too :(

File: site/fi_FI.all.json [hunk 4]
```diff
@@ -446,156 +446,153 @@
...
+  "wcag_aria_label_toggle_nav": { "other": "Navigointi päälle/pois" },
+  "wcag_aria_label_bundle_items": { "other": "Niputa kohteita" },
+  "wcag_aria_label_carousel": { "other": "Karuselli" },
+  "wcag_aria_label_page_collection": { "other": "Sivujen kokoelma" },
+  "wcag_aria_label_collection_list": { "other": "Kokoelmaluettelo" },
+  "wcag_aria_label_collection_slider": { "other": "Kokoelman liukusäädin" },
+  "wcag_aria_label_wishlist": { "other": "Toivelista" },
+  "wcag_aria_label_facebook": { "other": "Jaa Facebookissa" },
+  "wcag_aria_label_twitter": { "other": "Jaa Twitterissä" },
+  "wcag_aria_label_linkedin": { "other": "Jaa LinkedInissä" },
```
Comment (stajs) [inline site/fi_FI.all.json:469]: Ah, this is what I did too with the company names.

**Review by stajs — APPROVED** [review]
LGTM

File: site/es_ES.all.json (hunk 1 above)
Comment (brynyShift72) [reply site/es_ES.all.json:546]: Google Translate does have ENG->ES saying OK->OK, Translating it back and forth a few times gets you to `Okay/Bueno`
I'll update to be more specific. Looking around online makes me think that most Romance and Germanic languages are fine with using "OK", probably a Latin thing.

**Review by curtisadams28 — CHANGES_REQUESTED** [review]
`wcag_skip_links_footer` is no longer being used so should be removed.

**Review by curtisadams28 — APPROVED** [review]

File: site/es_MX.all.json (hunk 2 above)
Comment (brynyShift72) [reply site/es_MX.all.json:445]: I assume these are recognizable in other languages, maybe not Letterboxd

File: site/fi_FI.all.json (hunk 4 above)
Comment (smitsanghvi21) [reply site/fi_FI.all.json:469]: shouldn't the company name be in their original form?

Comment (stajs) [reply site/fi_FI.all.json:469]: I don't really know myself - I figured if we are using Google Translate as our bare minimum I was just going to trust it...

Comment (brynyShift72) [reply site/fi_FI.all.json:469]: its kind of hard to answer this one lol 
I think both yes and no, the company name by itself translates one to one, but adding the action such as "jaa/share on" will add the 'issa' on the end, I suspect there are some Finnish noun/ grammar rules coming into play, the Wiki for twitter seems to use both Twitter/Twitterissa, depending on context.

Comment (stajs) [reply site/fi_FI.all.json:469]: Also interesting the grammar rule has the `a` vs `ä`.

**Review by smitsanghvi21 — APPROVED** [review]

File: site/fi_FI.all.json (hunk 4 above)
Comment (brynyShift72) [reply site/fi_FI.all.json:469]: I'm tempted to go with the `issa` suffix as the `social_media_buttons_X` key values also include them, they were translated before these changes, so I suspect translated by a Finnish client?

**Pull request merged**